core/app/models/spree/inventory_unit.rb

```diff
@@ -79,6 +104,22 @@ def included_tax_total
       line_item.included_tax_total * percentage_of_line_item
     end
 
+    def empty?
```

Comment:
@Mafi88 Right, inventory unit quantity will not be 0 in any case. Infact, `singular?` and `empty?` methods are not used anywhere. Removing them.